consumer/multiplexer/multiplexer.go

```diff
@@ -230,18 +259,19 @@ reset:
 			if idx == inputCount {
 				return
 			}
-			m.sortedIns[idx].msg = value.Interface().(consumer.Message)
-			m.sortedIns[idx].msgOk = true
-		}
-		// At this point there is at least one message available.
-		inputIdx = selectInput(inputIdx, m.sortedIns)
-		// Block until the output reads the next message of the selected input
-		// or a stop signal is received.
-		select {
-		case <-m.stopCh:
-			return
-		case m.output.Messages() <- m.sortedIns[inputIdx].msg:
-			m.sortedIns[inputIdx].msgOk = false
+
```

Comment:
I would have made a stylistic change:
```
if isAtLeastOneAvailable {
    continue
}
```
So that all code that used to be inside `if !isAtLeastOneAvailable {}` can be one indentation interval less.